recipes/capytaine/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "1.0" %}
+
+package:
+  name: capytaine
+  version: {{ version }}
+
+source:
+  url: https://github.com/mancellin/capytaine/archive/v{{version}}.tar.gz
+  sha256: 072de801b50b4052277b5eb3d634734a11079b69789e5fb3bccb7363d199f637
+
+build:
+  skip: True  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+  host:
+    - python >= 3.6
```

This version constraint should be removed as there will be one build per Python version. The constraint comes from the `skip: True  # [py<36]` part of the recipe.
```suggestion
    - python
```

---

Ok, I removed the constraint.